frontend/src/Metamaps/Map/index.js

```diff
@@ -38,6 +38,11 @@ const Map = {
   init: function () {
     var self = Map
 
+    // prevent right clicks on the main canvas, so as to not get in the way of our right clicks
+    $('#wrapper').on('contextmenu', function (e) {
```

Comment:
as far as I can tell though, that id doesn't exist anymore... and when I just tested it, it doesn't prevent the default contextmenu